zk/arkworks/gramine/Cargo.toml

```
@@ -6,6 +6,7 @@ edition = "2021"
 [[bin]]
 name = "paramgen"
 path = "src/main.rs"
+required-features = ["paramgen"]
 
```

I think we need to specify `paramgen` as a feature -> 
```suggestion
required-features = ["paramgen"]

[features]
paramgen = ["penumbra-proof-setup"]
```

With that the following will work as expected -> 
```
$ cargo run --release --features=paramgen
```